Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -315,6 +315,64 @@ class AppStoreConnectService {
         .map(Model.BetaTester.init)
     }
 
+    func listBetaTestersForGroup(identifier: AppLookupIdentifier, groupName: String)
+        throws -> [Model.BetaTester]  {
+
+            var appId: String = ""
+
+            switch (identifier) {
+            case .appId(let id):
+                appId = id
+            case .bundleId(let bundleId):
+                let appsOperation = GetAppsOperation(options: .init(bundleIds: [bundleId]))
+                appId = try appsOperation.execute(with: requestor).compactMap(\.first).await().id
+            }
+
+            enum Error: LocalizedError {
+                case noGroupId
+                case multipleGroupIds
+
+                var errorDescription: String? {
+                    switch self {
+                    case .noGroupId:
+                        return "No group id exists"
+                    case .multipleGroupIds:
+                        return "More than 1 group ids returned"
+                    }
+                }
+            }
+
+            var groupIds: [String] = []
+            if !groupName.isEmpty {
+                groupIds = try ListBetaGroupsOperation(options: .init(appIds: [appId], names: [groupName], sort: nil))
```

Comment:
Can use `GetBetaGroupOperation` here for getting only one ID. A refactor need to be done to `GetBetaGroupOperation` to achieve this. (let it take bundle or appId instead of `App`)